src/rules/noUndefinedTypes.js

```diff
@@ -206,11 +208,11 @@ export default iterateJsdoc(({
     }) => {
       if (type === 'JsdocTypeName') {
         const structuredTypes = structuredTags[tag.tag]?.type;
-        if (!allDefinedTypes.has(value) &&
+        if (!disableReporting && !allDefinedTypes.has(value) &&
```

Comment:
Won't this risk marking an undefined type as used? I don't know if that's a problem or just a no-op.

Reply:
Hmmm. Good catch. I think it is a problem. However, I was considering though whether someone just wants all used types to be trusted as used. Not sure if that deserves its own config too, or whether I should solely put the `disableReporting` inside the `if`.

Reply:
> put the `disableReporting` inside the `if`.

I think that would be the simplest and safe bet 👍